### DISCUSSION TITLE
Breezewiki

### DISCUSSION THREAD

**Discussion by kingofcube** [catppuccin/catppuccin#2226] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

just make it like the other userstyles (different flavours, all of the accents, blah blah blah)

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Hey :wave: 

Please do not pad out the description with information that is not directly relevant to the question in the heading, **"just make it like other userstyles"** is not relevant here for that heading. A number of community members have also flagged that the wording in the description comes across as dismissive of the effort put into making a userstyle.

Can you update that section with a link to the website that you're wanting themed? If you're unsure about how to theme it, it's okay to admit that.

**Reply by kingofcube** [reply]
ok
how do i edit it
